app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -274,6 +289,68 @@ public void exportData(View view) {
         }
     }
 
+    public void exportTest(View view) {
+        scheduleSingleDataUpload();
+    }
```

Comment:
scheduleSingleDataUpload wordt niet hergebruikt en is ook public, wat deze functie overbodig maakt.
Als er een goede reden is om deze functie te laten blijven bestaan moet-ie wel een mooiere naam krijgen :) + dan kan scheduleSingleDataUpload  private worden

ctrl + c...

Reply:
deze functie is tijdelijk. Hij heeft een View als variabele. Daardoor kan hij makkelijk vanuit de GUI via onclick worden  aangeroepen. DIt is een tijdelijke functie. Deze moet gedurende de verdere intergratie worden verwijderd